linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/HttpConfigTest.scala

```diff
@@ -1,10 +1,13 @@
 package io.buoyant.linkerd.protocol.http
 
+import com.fasterxml.jackson.databind.exc.InvalidDefinitionException
+import com.twitter.conversions.StorageUnitOps._
 import com.twitter.finagle.buoyant.Dst
+import com.twitter.finagle.http.param.{FixedLengthStreamedAfter, Streaming}
 import com.twitter.finagle.http.{Method, Request}
 import com.twitter.finagle.{Dtab, Path, Stack}
 import com.twitter.util.Future
-import io.buoyant.config.Parser
+import io.buoyant.config.{ConflictingStreamingOptions, Parser}
 import io.buoyant.linkerd.{IdentifierInitializer, RouterConfig}
 import io.buoyant.linkerd.protocol.{HttpConfig, HttpIdentifierConfig, HttpInitializer}
 import io.buoyant.router.Http
```

Comment:
The `com.twitter.conversions.StorageUnitOps._`, `FixedLengthStreamedAfter` and `ConflictingStreamingOptions` seem to be unused, can we just remove them ?